cpp/log/database_test.cc

```diff
@@ -47,7 +47,12 @@ typedef testing::Types<FileDB<cert_trans::LoggedCertificate>,
 
 typedef Database<cert_trans::LoggedCertificate> DB;
 
+template <class T>
+class DBTestDeathTest : public DBTest<T> {
```

Comment:
I tried it here, it works for me? Had to replace the whole templated class definition with the snippet I gave, but was fine. Oh well, no big deal, keep it like this, then. :smiley: